website/docs/components/stepper/nav/partials/specifications/states.md

```diff
@@ -0,0 +1,3 @@
+## States
+
+![Stepper Nav states](/assets/components/stepper/nav/stepper-nav-states.png)
```

Comment:
```suggestion
![Non-interactive Stepper Nav steps only have one state. Interactive Stepper Nav steps have multiple: completed steps have default, active, hover, focus states, the current step has default and focus states, and inactive steps only have a default state because they are not interactive.](/assets/components/stepper/nav/stepper-nav-states.png)
```